tests/integration/clientv3/maintenance_test.go

```diff
@@ -156,6 +157,24 @@ func TestMaintenanceSnapshotCancel(t *testing.T) {
 
 	// reading snapshot with canceled context should error out
 	ctx, cancel := context.WithCancel(context.Background())
+
+	// Since http2 spec defines the receive windows's size and max size of
```

Comment:
Nice. Thank you. 

Could you, please, move it to a separate helper method as it shared piece of logic, e.g.

func populateWithData(t testing.T, cluster *Cluster, numKeys int, valueSize int) {
...
}

Reply:
+1

Reply:
Sure! Updated. PTAL Thanks!